File: Crossplatform/c_sharp_version/Middleware/domain/ChatLogs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Middleware.domain
{
    [System.Serializable]
    public class ChatLogs
    {
        public List<ChatMessage> chatMessages = new List<ChatMessage>();
        public ChatLogs() 
        {

        }
        public ChatLogs(List<ChatMessage> messages)
        {
            this.chatMessages = messages;
        }
        public void addLog(ChatMessage message)
        {
            chatMessages.Add(message);
        }
        
    }
}

File: Crossplatform/java_network_FINAL1/Middleware/src/main/java/domain/ChatLogs.java
package domain;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class ChatLogs implements Serializable {
    public List<ChatMessage> chatMessages = new ArrayList<ChatMessage>();
    public ChatLogs() {}
    public ChatLogs(List<ChatMessage> messages){
        this.chatMessages = messages;
    }
    public void addLog(ChatMessage message)
    {
        chatMessages.add(message);
    }
}
